src/api/v1/db/oracledb/ingredients-dao.js

```diff
@@ -15,6 +17,70 @@ const ingredientsColumnNames = {
   notes: 'NOTES',
 };
 
+/**
+ * convert an output bind param name like `nameOut` to a
+ * property name like `name` by removing 'Out' at the end of the string
+ * @param {string} outBindParamName
+ * @returns {string} the property name
+ */
+const outBindParamToPropertyName = (outBindParamName) => outBindParamName.slice(0, -3);
+
+/**
+ * A list of the columns in the INGREDIENTS table joined by commas
+ * @constant
+ * @type {string}
+ */
+const ingredientColumns = _.map(ingredientsProperties,
+  (propertyValues, property) => ingredientsColumnNames[property])
+  .join(', ');
+
+/**
+ * The full list of valid ingredient attributes for a POST request,
+ * prepended with `:` and joined with commas
+ * @constant
+ * @type {string}
+ */
+const ingredientValues = _.map(ingredientsProperties, (propertyValues, property) => `:${property}`)
```

Comment:
Same, you didn't use values of the object at all so I think you can improve it by using `_.keys` as well.